app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
```

Comment:
Do not forget to indicate all acceptable types in the type annotation in the future
```suggestion
    def __init__(self, km: int | float) -> None:
```